scripts/get-service-editing-urls.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+"""
+Get service editing urls for each page in DOS service flow.
+
+"""
+from dmutils import content_loader
+
+
+def main():
+    lots = ['digital-outcomes', 'digital-specialists', 'user-research-studios',
+            'user-research-participants']
+    loader = content_loader.ContentLoader('../digitalmarketplace-frameworks')
+    loader.load_manifest('digital-outcomes-and-specialists', 'services', 'edit_submission')
+    manifest = loader.get_manifest('digital-outcomes-and-specialists', 'edit_submission')
+
+    url = '/suppliers/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>'
+    url = '/suppliers/frameworks/{}/submissions/{}/{}'
```

Comment:
If you want keyword substitution this could be
`url = '/suppliers/frameworks/{framework_slug}/submissions/{lot_slug}/{service_id}'`
and then format's keyword substitution can be used
`url.format(framework_slug='whatever', lot_slug='something', service_id='5432')`